### ISSUE TITLE
More layer and unit details

### ISSUE THREAD

**Issue by tonio73** [tonio73/dnnviewer#19]
Missing layer information in the viewer:
- loss (_training_)
- activation at output (_output_)
- regularization (L1/L2, kernel/bias/activity) (_training_)
- dropout at input  (_training_)
- batch normalization at input (_input_)
- Max/Average pooling at output (_output_)

Missing unit information in the viewer:
- bias/intercept (.use_bias, .bias) (_training_)
- input shape (.input.get_shape()) (_structure_)
- output shape (.ouput_shape[1:]) (_structure_), take into account for pooling and flatten
- padding (Convo) (.padding) (_structure_)

Later:
- Layer metrics

Implementation:
- foreach layer and unit create a decorator dictionary with following attributes containing lists of strings
- attributes : input, structure, output, training
- fill this structure in bridge.tensorflow.keras_extract_sequential_network() taking into account for the fact that the layer may not exist (previous layers like dropout, batchnorm) or may already exist (layers at output like pooling, activation)

**Comment by tonio73** [comment]
Depends on #22
